content/blog/2023/2023-09-07-introduction-to-kDoc

```diff
@@ -0,0 +1,111 @@
+---
+title: "Introduction to KDoc"
+categories: ["Kotlin"]
+date: 2023-09-07 00:00:00 +1100 
+authors: [ezra]
+excerpt: "In this tutorial, we'll discuss KDoc which is a documentation langunage for Kotlin code."
+image: images/stock/0104-on-off-1200x628-branded.jpg
+url: introduction-to-kDoc
+---
+
+In this tutorial, we'll discuss all that entails <em>KDoc</em> in Kotlin. [KDoc](https://kotlinlang.org/docs/kotlin-doc.html#links-to-elements) is simply language used to document code written using Kotlin specifically. KDoc allows us to provide documentation comments for classes, functions, properties and other elements in our code. It's the same as <em>Javadoc</em> which is used to document JAVA language. Essentially, <em>KDoc</em> combines syntax in <em>Javadoc </em>for the block tags and markdown for inline markup.
+## 2. KDoc Syntax</h2>
+Same as <em>javadoc, </em>KDoc comments usually start with /** and end with */.
+
+Let’s see an example of KDoc:
+```groovy
+/**
+ * Calculates the sum of two numbers.
+ *
+ * @param a The first number.
+ * @param b The second number.
+ * @return The sum of the two numbers.
+ */
+fun sum(a: Int, b: Int): Int {
+    return a + b
+}
+```
+KDoc in our example is written above our <em>sum</em> function. In this case, KDoc explains what task our function performs and also documents the parameters <em>a</em> and <em>b</em> which the function takes inclusive of the expected return value.
+## 3. Tags Supported by Kdoc
+Tags that are supported by KDoc are grouped into two categories:
+<ul>
+ 	<li>Block tags</li>
+ 	<li>Inline markup</li>
+</ul>
+## 4. Block Tags
+Block tags are used to provide documentation for larger sections of code or to describe multi-line content within KDoc.They are usually placed on separate lines.
+
+These are the block tags supported by KDoc:
+
+**@param name**:This tag is used to document a value parameter of a function.
+
+**@return**: Used to document the return value of a function.
+
+**@constructor**: Used to document the primary constructor of a class.
+
+**@receiver**: Documents the receiver of an extension function.
+
+**@property name**: This tag is used to document the property of a class that has the specified name.
```

Comment:
It's not quite clear that "name" is a parameter here. Can you put it in brackets like <name>? Similar for other parameters in other examples.